SEAN MAVHIMA.  REG: 223025636

```diff
@@ -0,0 +1,405 @@
+STUDENT REG NUMBER: 223025636
+JAVA PROGRAMMING ASSIGNMENT
+
+QUESTION 1:
```

Comment:
Question 1 was done in Intellij IDEA 2024.1
Question2 was done in Netbeans IDE 23